src/components/LandingPage/index.tsx

```diff
@@ -1,33 +1,63 @@
+import { useState } from "react";
 import Image from "next/image";
+import { CircleNotch } from "phosphor-react";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
-import { Header } from "./Header";
-import imgHome from "@/app/assets/img/imagem-site.png";
+import imgHome from "@/app/assets/img/portal.png";
+import { Header } from "@/components/LandingPage/Header";
+import { Button } from "@/components/Button";
 
 export function LandingPage() {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Só uma observação mesmo, apesar da nomenclatura isLoading ser comum de encontrar, ela é genérica e n diz muito para o dev qual o contexto do loading, por exemplo, se nesse contexto ela se chamasse isRedirectingToLogin, ficaria mais rápido pro dev identificar a funcionalidade dela. Isso claro, se ela n for utilizada em outros lugares.